web/src/pages/profile.tsx

```diff
@@ -329,7 +329,7 @@ const ProfilePage = (props: Props): ReactElement => {
     if (!business) return false;
     return (
       (profileData.industryId === "real-estate-investor" || profileData.sectorId === "real-estate") &&
-      (profileData.operatingPhase === "UP_AND_RUNNING" || profileData.operatingPhase === "GUEST_MODE_OWNING")
+      (profileData.operatingPhase === "UP_AND_RUNNING" || profileData.businessPersona === "OWNING")
```

Comment:
Updated from operating phase to check for business persona. The data can be in 3 states:

1. Up and running poppy (captured by first half of this conditional)
2. Guest mode Oscar (captured by the second half of this conditional)
3. Non-guest mode Oscar (now captured by the second half of this conditional)